README.md

```diff
@@ -1,25 +1,96 @@
 # AtlasTests
 
-To start you have to install cucumber plugin to IntelliJ Idea: https://plugins.jetbrains.com/plugin/7212-cucumber-for-java
-Please take attention on version of your Idea, to download correct version of plugin. (Help -> About)
+### Common and configuration
 
 Clone this project to your computer.
 
-Create file "application.properties" with parameters:
-login
-password
-downloadpath
+You have to set correct parameters in *.feature files (blue color), e.g. you have to set existing cohorts
+or concept sets, also you have to set existing data sources with correct names (as on UI). In another
+case tests will have "failed" status.
+
+Create file **application.properties** in _src_ directory with parameters:
+
+- login
+- password
+- downloadpath
+- test.mode  
+- test.browser 
+- link
+- e.g.
+
+
+_login_ and _password_ is Atlas account for tests
+
+_downloadpath_ is a directory to save files (Note: directory should be exist)
+
+_test.mode_ local or remote
+
+_test.browser_
+
+_link_ is link to test environment
+
+
+
+You can run tests in a local browser or remote Selenium Server
+
+- for **local** mode is specifying path to browser driver, like _src/chromedriver.exe_
+- for **remote** mode is specifying URL for Selenium Server, like _http://127.0.0.1:4444/wd/hub_ 
+(note: instruction for setup and configure Selenoide described in section "Setup and configure Selenoide") 
+
+**Requirements for running test**
+
+I need installed JDK and Apache Maven 
```

Comment:
`I need`? Why not passive form? e.g. `It is required to have blah-blah-blah`

Reply:
It was written by M. Ivanko. I didn't check it. I'll fix it.